addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -604,54 +619,30 @@ var FileWidget = SearchableMediaWidget.extend({
             // limited by bandwidth.
             uploadMutex.exec(function () {
                 return utils.getDataURLFromFile(file).then(function (result) {
-                    var params = {
-                        'name': file.name,
-                        'data': result.split(',')[1],
-                        'res_id': self.options.res_id,
-                        'res_model': self.options.res_model,
-                    };
-                    if (self.quickUpload) {
-                        params['width'] = self._computeOptimizedWidth();
-                        params['quality'] = 80;
-                    } else {
-                        params['width'] = 0;
-                        params['quality'] = 0;
-                    }
                     return self._rpc({
                         route: '/web_editor/attachment/add_data',
-                        params: params,
+                        params: {
```

Comment:
should we have a garbage collecting of picture when you click on save to remove unsuded picture uploaded during this edition ? 

If you drop 2 pictures on page, before to find the right... you will have 6 attachment for 1 (+1original) picture only.

Reply:
long discussion, no aggreement with qsm.

So lets grow the db meanwhile ;) we never know.. that somebody want to use odoo website as backup server of photo before to push it in instagram ;)

Reply:
I said that, as it has to be done for unsplash, and given the current state of the media dialog (no multi upload, no sorting, advanced seach etc), I am ok with a generic solution to garbage collect all of them :wink: In this PR or not, same for me.

In a perfect media dialog, I'd like the _upload_ button to mean _upload_ and not _upload and delete if not used_ yes, but not really important :+1: